packages/marko-web-identity-x/routes/login.js

```diff
@@ -2,90 +2,56 @@ const gql = require('graphql-tag');
 const { asyncRoute } = require('@base-cms/utils');
 
 const query = gql`
-  query AppUser($input: AppUserQueryInput!) {
-    appUser(input: $input) {
+  query LoginCheckAppUser($email: String!) {
+    appUser(input: { email: $email }) {
       id
       email
-      givenName
-      familyName
-      organization
-      organizationTitle
-      countryCode
-      regionCode
-      postalCode
     }
   }
 `;
 
 const createUser = gql`
-  mutation CreateAppUser($input: CreateAppUserMutationInput!) {
-    createAppUser(input: $input) {
+  mutation LoginCreateAppUser($email: String!) {
+    createAppUser(input: { email: $email }) {
       id
       email
-      givenName
-      familyName
-      organization
-      organizationTitle
-      countryCode
-      regionCode
-      postalCode
     }
   }
 `;
 
 const sendLoginLink = gql`
-  mutation SendLoginLink($input: SendAppUserLoginLinkMutationInput!) {
+  mutation LoginSendLoginLink($input: SendAppUserLoginLinkMutationInput!) {
     sendAppUserLoginLink(input: $input)
   }
 `;
 
-const getMissingFields = (user, requiredFields) => requiredFields.filter(field => !user[field]);
-
 module.exports = asyncRoute(async (req, res) => {
   const { identityX, body } = req;
-  // @todo this could come from the MW config??
   const {
-    user,
+    email,
     authUrl,
-    requiredFields = [],
     redirectTo,
   } = body;
-  const input = { email: user.email };
-  const variables = { input };
+  const variables = { email };
```

Comment:
I don't see a change to the `createAppUser` mutation in this PR -- shouldn't this be `{ input: { email } }`? Also we've generally been keeping the standard of using `input` at the root, should this even be changed?

Reply:
It still is root input, and `createAppUser` didn't change. This is a _local_ variable def (see the top of the file)